go.mod

```diff
@@ -1,7 +1,6 @@
 module github.com/ceph/ceph-csi
 
 go 1.22
-
 toolchain go1.22.2
```

Comment:
```bash
$]cat go.mod |grep -i "1.22"
go 1.22
toolchain go1.22.2
	code.cloudfoundry.org/gofileutils => github.com/cloudfoundry/gofileutils v0.0.0-20170111115228-4d0c80011a0f
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f ceph-csi $]go mod tidy
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f ceph-csi $]git diff
diff --git a/go.mod b/go.mod
index 076fc38c7..92ce83ca3 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,7 @@
 module github.com/ceph/ceph-csi
 
-go 1.22
+go 1.22.0
+
 toolchain go1.22.2
 
 require (
[🎩︎]mrajannaceph-csi $]go version
go version go1.22.2 linux/amd64
```

@nixpanic when i ran `go mod tidy` above is getting reverted back.

Reply:
What version of Go do you use?

Reply:
Strange, what do you recommend then?

Have the minor version included, as it might get added by some commands again anyway?
```
go 1.22.0

toolchain go1.22.2
```

Reply:
we can have both or drop toolchain, if we are going with both lets keep the version same in both (which is nothing both dropping toolchain internally)

Reply:
Now without `toolchain`. I guess `go mod tidy` added that at one point. Maybe it is not required anymore?